docs/src/index.md

```diff
@@ -42,8 +42,14 @@ julia> BandedMatrix(Ones(5,5), (1,2))
            1.0  1.0  1.0
                 1.0  1.0
 ```
-
-
+To create a banded matrix of a given size with constant bands (such as the classical finite difference approximation of the two-dimensional Laplacian on the unit square [0,1]×[0,1]), you can use the following:
```

Comment:
The code below does the one-dimensional Laplacian on the the unit interval `[0,1]`.

Reply:
Fair enough, without reordering afterwards this is not useful. I'll change it back.